fury/tests/test_material.py

```diff
@@ -29,65 +24,71 @@ def test_manifest_pbr_vtk_great_than_9():
-
-        test_actor = actor.square(np.array([[0, 0, 0]]), directions=(0, 0, 0),
-                                  colors=(0, 0, 1))
-
-        scene.add(test_actor)
-
-        # Test basic actor
-        window.record(scene, out_path=tmp_fname, size=(200, 200),
-                      reset_camera=True)
-        npt.assert_equal(os.path.exists(tmp_fname), True)
-        ss = load_image(tmp_fname)
-        actual = ss[100, 100, :] / 1000
-        desired = np.array([0, 0, 255]) / 1000
-        npt.assert_array_almost_equal(actual, desired, decimal=2)
-        actual = ss[40, 40, :] / 1000
-        npt.assert_array_almost_equal(actual, desired, decimal=2)
-
-        # Test default parameters
-        material.manifest_pbr(test_actor)
-        window.record(scene, out_path=tmp_fname, size=(200, 200),
-                      reset_camera=True)
-        npt.assert_equal(os.path.exists(tmp_fname), True)
-        ss = load_image(tmp_fname)
-        actual = ss[100, 100, :] / 1000
-        desired = np.array([66, 66, 165]) / 1000
-        npt.assert_array_almost_equal(actual, desired, decimal=2)
-        actual = ss[40, 40, :] / 1000
-        desired = np.array([40, 40, 157]) / 1000
-        npt.assert_array_almost_equal(actual, desired, decimal=2)
-
-        # Test roughness
-        material.manifest_pbr(test_actor, roughness=0)
-        window.record(scene, out_path=tmp_fname, size=(200, 200),
-                      reset_camera=True)
-        npt.assert_equal(os.path.exists(tmp_fname), True)
-        ss = load_image(tmp_fname)
-        actual = ss[100, 100, :] / 1000
-        desired = np.array([0, 0, 155]) / 1000
-        npt.assert_array_almost_equal(actual, desired, decimal=2)
-        actual = ss[40, 40, :] / 1000
-        desired = np.array([0, 0, 153]) / 1000
-        npt.assert_array_almost_equal(actual, desired, decimal=2)
-
-        # Test metallicity
-        material.manifest_pbr(test_actor, metallicity=1)
-        window.record(scene, out_path=tmp_fname, size=(200, 200),
-                      reset_camera=True)
-        npt.assert_equal(os.path.exists(tmp_fname), True)
-        ss = load_image(tmp_fname)
-        actual = ss[100, 100, :] / 1000
-        desired = np.array([0, 0, 255]) / 1000
-        npt.assert_array_almost_equal(actual, desired, decimal=2)
-        actual = ss[40, 40, :] / 1000
-        desired = np.array([0, 0, 175]) / 1000
-        npt.assert_array_almost_equal(actual, desired, decimal=2)
+    # with TemporaryDirectory() as out_dir:
+    # tmp_fname = os.path.join(out_dir, 'tmp_img.png')  # Tmp image to test
+
+    scene = window.Scene()  # Setup scene
+
+    test_actor = actor.square(np.array([[0, 0, 0]]), directions=(0, 0, 0),
+                              colors=(0, 0, 1))
+
+    scene.add(test_actor)
+
+    # Test basic actor
+    # window.record(scene, out_path=tmp_fname, size=(200, 200),
+    #              reset_camera=True)
+    ss = window.snapshot(scene, size=(200, 200))
```

Comment:
@Garyfallidis thank you for the replacement. All the VTK's PBR tests seem to be working.